bundles/replication/src/main/java/com/adobe/cq/commerce/demandware/replication/transport/WebDAVTransportPlugin.java

```diff
@@ -76,9 +74,6 @@
     @Property(label = "WebDAV user password")
     private static final String WEBDAV_PASSWORD = "webdav.password";
 
-    @Reference
-    DemandwareClient demandwareClient;
-
     private String webDavEndpoint;
     private String webDavUser;
```

Comment:
we have to make that a factory when it has local variables. or we move the credentials to the demandware-client

Reply:
agreed to put it into the client